libraries/AC_CustomControl/AC_CustomControl_Backend.h

```diff
@@ -0,0 +1,34 @@
+#pragma once
+
+#include "AC_CustomControl.h"
+
+#if CUSTOMCONTROL_ENABLED
+
+class AC_CustomControl_Backend
+{
+public:
+    AC_CustomControl_Backend(AC_CustomControl& frontend, AP_AHRS_View& ahrs, AC_AttitudeControl& atti_control, AP_Motors& motors, float dt) :
```

Comment:
In other places we call the AC_AttitudeControl argument "att_control" (no extra "i")

Reply:
Change it to att_control